src/components/PriceInfo/index.tsx

```diff
@@ -126,10 +124,10 @@ const Index = ({
             />
           ) : null}
 
-          {priorityFeeInSOL > 0 ? (
+          {priorityFee > 0 ? (
             <div className="flex items-center justify-between text-xs">
               <div className="text-white/30">Priority Fee</div>
-              <div className="text-white/30">{new Decimal(priorityFeeInSOL).toString()}</div>
+              <div className="text-white/30">{new Decimal(priorityFee).toString()} SOL</div>
```

Comment:
Added the postfix unit because previously without unit is quite confusing

![image](https://github.com/chiew256/terminal/assets/73978919/bd978955-f98a-4e53-b8ef-b7287b483c2f)